app/graphql/mutations/destroy_task.rb

```diff
@@ -0,0 +1,16 @@
+module Mutations
+
+  class DestroyTask < BaseMutation
+    argument :input, Types::Inputs::DestroyTaskInput, required: true
+
+    type Types::Payloads::TaskPayload
+
+    def resolve(input:)
+      @task = Task.find(input.id)
+      @project = Project.find(input.project_id)
```

Comment:
You don't need instance variable

```suggestion
      task = Task.find(input.id)
      project = Project.find(input.project_id)
```